Packages/SystemServices/BannerService/BannerSetupService.swift

```diff
@@ -46,4 +46,10 @@ public struct BannerSetupService: Sendable {
             NewBanner.accountBlockedMultiSignature(walletId: walletId, chain: chain)
         ])
     }
+    
+    public func setupBuyCryptoBanner(walletId: WalletId) throws {
+        try store.addBanners([
+            NewBanner.buyCrypto(walletId: walletId)
```

Comment:
```suggestion
            NewBanner.buyCrypto(walletId: walletId)
```

it's not related to buy crypto, this is onboarding flow and not domain driven functionality